### PR TITLE
Assignment 2 News site

### PR DESCRIPTION
**Pull request by gtine**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,188 @@
...
+  padding-bottom: 40px;
+  }
+
+.big-news-section img {
+  width: 40%;
+  }
+
+.big-news-section div{
+  padding:0px 30px 0px 0px;
+  border-bottom:none;
```
Comment (JennieDalgren) [inline code/style.css:171]: this is already set in the media query above, line 131. No need to add it again as the rule above applies here as well

**Review by JennieDalgren — CHANGES_REQUESTED** [review]
Good job with this project. 
It looks really nice.
Although, the code needs some tidying up. Make sure to fix the indentation.

Take a look at[ our code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e).
We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

Nice job with shadow effect on the cards and you're already mastering the flexbox!
Keep up the good work 🚀

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,90 @@
...
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+      </head>
+      <body>
+
```
[the remark this answers is not in the document]
Comment (gtine) [reply code/index.html:10]: done :)

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,188 @@
 html {
-  background: #0025ff;
-  color: #fff;
+    background: rgb(134, 128, 128)
+    color:black;
 }
+
+body {
+  margin:0px;
```
[the remark this answers is not in the document]
Comment (gtine) [reply code/style.css]: done :)

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,188 @@
...
+  border-bottom: solid rgb(46, 7, 7);
+}
+
+.big-news-section img {
+  width: 100%;
+  box-shadow: 5px 10px rgb(46, 7, 7);
+}
+
+.big-news-section div {
+  padding-bottom: 20px;;
```
[the remark this answers is not in the document]
Comment (gtine) [reply code/style.css]: done :)

File: code/style.css (hunk 1 above)
Comment (gtine) [reply code/style.css:180]: I did remove some of the doubles, although I left the padding part as it was different

**Comment by gtine** [comment]
Hi Jennie, thanks for your comments. I have made the requested changes but had some difficulties with pushing again, it said error cause the branches didnt match. I pulled and then pushed again and then it worked. I hope I have done it correctly now?  Should I close now or how do I move on? Thanks alot. Tine

File: code/index.html [hunk 5]
```diff
@@ -2,21 +2,85 @@
...
+<header class="header1">
+  <div class="hamburger">
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+
+    <img class="logo-pic" src="qllogo.png" alt="logo">
+    <h1 class="qtimes">Quarantine Times</h1>
```
Comment (HIPPIEKICK) [inline code/index.html:20]: Since these two are "siblings" to thi div and the nav, they should not be indented

File: code/index.html [hunk 6]
```diff
@@ -2,21 +2,85 @@
...
+    <a href="">Science</a> |
+    <a href="">Style</a> |
+    <a href="">Daily</a>
+  </nav>
+</header>
+       
+<section class="big-news-section">
+<!-- The big news content -->
+    <img src="kneading.jpg" alt="kneadingbread">
+  <div>
```
Comment (HIPPIEKICK) [inline code/index.html:33]: Same with these two, they are siblings. Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap about indentation :)

**Pull request closed without merging**